### DISCUSSION TITLE
Create video tutorials for the new users

### DISCUSSION THREAD

**Discussion by shotarok** [polyaxon/polyaxon#1459] [Ideas]
We have been using Polyaxon for years as the ML Platform team. One feedback we got from ML engineers is the onboarding cost. One idea to help the onboarding is to create video tutorials for new users.

**Comment by md-mq** [comment]
Hi @shotarok ,

I already started some work around some recurrent use-cases that I see in demos: https://www.youtube.com/channel/UCbzCxshQWTxCZPWZwgM0ePw
More short content around specific use-cases is under work.
If you have specific use-cases that you would like to see and share with the rest of your team as an educational or on-boarding material, please let me know.

**Reply by shotarok** [reply]
I didn't know the demo videos. Thanks! I really like them. Yes, when I find additional tutorials we'd like to see, I'll post them here.

**Comment by md-mq** [comment]
Another idea that I was thinking about is starting a bi-weekly or a monthly community call, where anyone can join and we can either demo specific aspects of the platform or answer questions.

**Reply by shotarok** [reply]
That's also an interesting idea. In our company's case, we have teams in different time zones. So we don't have a good time for all ML engineers to come together at the same time. Therefore, the videoes demos would work for us. Everyone can learn polyaxon asynchronously 🙂
